src/pudl/metadata/resources/ferc1.py

```diff
@@ -537,14 +537,21 @@
         "field_namespace": "ferc1",
     },
     "plants_small_ferc1": {
-        "description": (
-            "Generating plant statistics for steam plants with less than 25 MW "
-            "installed nameplate capacity and internal combustion plants, gas "
-            "turbine-plants, conventional hydro plants, and pumped storage plants with "
-            "less than 10 MW installed nameplate capacity. As reported on FERC Form 1 "
-            "Schedule 410 (pages 410-411), and extracted from the FERC Visual FoxPro "
-            "database table f1_gnrt_plant."
-        ),
+        "description": """The generating plant statistics for steam plants with less
+than 25 MW installed nameplate capacity as well as internal combustion
+plants, gas turbine-plants, conventional hydro plants, and pumped storage plants with
+less than 10 MW installed nameplate capacity. As reported on FERC Form 1 Schedule 410
+(pages 410-411) and extracted from the FERC Visual FoxPro and XBRL. See our
+:py:const:`pudl.extract.ferc1.TABLE_NAME_MAP_FERC1` for linkages to the raw tables.
```

Comment:
Note that RST in these strings will get rendered in our documentation (Read The Docs), but will show up as-is in Datasette.

Reply:
mmmm annoying, I will remove the :py:const: part then